samples/test_azure_event_hubs_receive.py

```diff
@@ -236,6 +236,32 @@ def test_event_hubs_iter_receive_sync(live_eventhub_config):
     receive_client.close()
 
 
+def test_event_hubs_shared_connection(live_eventhub_config):
+    uri = "sb://{}/{}".format(live_eventhub_config['hostname'], live_eventhub_config['event_hub'])
+    sas_auth = authentication.SASTokenAuth.from_shared_access_key(
+        uri, live_eventhub_config['key_name'], live_eventhub_config['access_key'])
+    source = "amqps://{}/{}/ConsumerGroups/{}/Partitions/".format(
+        live_eventhub_config['hostname'],
+        live_eventhub_config['event_hub'],
+        live_eventhub_config['consumer_group'])
+
+    with uamqp.Connection(live_eventhub_config['hostname'], sas_auth, debug=False) as conn:
+        partition_0 = uamqp.ReceiveClient(source + "0", debug=False, auth=sas_auth, timeout=3000, prefetch=10)
+        partition_1 = uamqp.ReceiveClient(source + "1", debug=False, auth=sas_auth, timeout=3000, prefetch=10)
+        partition_0.open(connection=conn)
+        partition_1.open(connection=conn)
+
+        try:
+            messages_0 = partition_0.receive_message_batch(1)
```

Comment:
From my comment in the other pr, should send before receiving to not just assume messages were sent from other tests.

Reply:
Fixed, added sending